app-nginx/__main__.py

```diff
@@ -0,0 +1,37 @@
+"""A pulumi program to deploy wordpress to kubernetes via helm on the asterion rpi server."""
```

Comment:
Refer to other comment, these references to asterion infra should be generic as we will be testing these app deployments against cloud infra in addition to our raspberry pi stacks.